ui/appui-react/src/appui-react/statusbar/StatusBarComposer.tsx

```diff
@@ -541,14 +542,80 @@ export function StatusBarComposer(props: StatusBarComposerProps) {
     >
       <StatusBarSpaceBetween className={mainClassName}>
         <StatusBarLeftSection className={leftClassName}>
-          {leftItems}
+          {leftItems.map((item, index) => (
```

Comment:
Can this be refactored/simplified, looks like spaghetti.

Reply:
Yeah, my bad. I forgot that in previous PRs I already sorted the status bar items. I think now it is pretty clean.